.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
```

Comment:
i think it may be helpful to have subcategory templates for data quality issues, issues relating to ingestion of documents or data not being parsed properly.

Reply:
Sure, I'll create them. It's just that I did not find the temaplte contents itself needing to change, only the label.